examples/templates/vue/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "vue",
```

Other templates have naming convention:
```suggestion
  "name": "vue-template",
```